software/control/filterwheel.py

```diff
@@ -0,0 +1,64 @@
+import os
+import time
+
+from control._def import *
+
+class SquidFilterWheelWrapper:
+
+    def __init__(self, navigationController = None):
```

Comment:
Is it actually valid for navigationController to be None?  Or should we check below that it isn't and throw?

Reply:
@veerwang please fix this. `navigationController` is a required argument

Reply:
Sure. I would do it.